### PR TITLE
Blogs + fix routes

### PR DESCRIPTION
**Pull request by intern-tieumomo**
## Related Tickets:
- [#28878](https://edu-redmine.sun-asterisk.vn/issues/28878)
- [#28886](https://edu-redmine.sun-asterisk.vn/issues/28886)

## WHAT this PR does?

- View blog, comment

## Notes Impacted Areas:
- N/A.

## Attachments:
screencapture-localhost-8000-blogs-2020-10-16-13_50_02
screencapture-localhost-8000-blogs-7-2020-10-16-13_49_39

### CODE REVIEW

**Comment by intern-tieumomo** [comment]
ready

File: app/Http/Controllers/BlogController.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+class BlogController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $posts = Post::orderBy('id', 'desc')->paginate(5);
```
Comment (huyenttm-0901) [inline app/Http/Controllers/BlogController.php]: đưa số vào config

File: app/Http/Controllers/CommentController.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+namespace App\Http\Controllers;
+
+use App\Models\Comment;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class CommentController extends Controller
+{
+    public function comment(Request $request)
```
Comment (huyenttm-0901) [inline app/Http/Controllers/CommentController.php]:
```suggestion
    public function create(Request $request)
```

File: resources/views/blogs.blade.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            {{ trans('text.blog.blogs') }}
+        </h2>
+    </section>
+
+    <section class="bgwhite p-t-60">
+        <div class="container">
+            <div class="row justify-content-center">
+                <div class="col-md-12 col-lg-9 p-b-75">
+                    <div class="p-r-50 p-r-0-lg">
+                        @foreach($posts as $post)
```
Comment (huyenttm-0901) [inline resources/views/blogs.blade.php]: convention

File: resources/views/blogs.blade.php [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+                                            <span class="m-l-3 m-r-6">|</span>
+                                        </span>
+
+                                        <span>
+                                            {{ count($post->comments) }} {{ trans('text.blog.comments') }}
+                                        </span>
+                                    </div>
+
+                                    <p class="p-b-12">
+                                        {{ Str::limit($post->content, 200, ' ...') }}
```
Comment (huyenttm-0901) [inline resources/views/blogs.blade.php]: 200 đưa vào config

**Comment by intern-tieumomo** [comment]
ready

**Pull request merged**